yagpdb_docker/docker-compose.dev.yml

```diff
@@ -1,48 +1,56 @@
-version: '3'
-
-volumes:
-  db:
-  redis:
-  cert_cache:
-  soundboard:
-
-networks:
-  default:
-
-services:
-  app:
-    build:
-      # We change context so that we can copy the local repo in during
-      # development
-      context: ../
-      dockerfile: yagpdb_docker/Dockerfile
-    restart: unless-stopped
-    depends_on:
-      - redis
-      - db
-    networks:
-      - default
-    volumes:
-      - cert_cache:/app/cert
-      - soundboard:/app/soundboard
-    env_file:
-      - app.env
-
-  redis:
-    image: redis
-    restart: unless-stopped
-    networks:
-      - default
-    volumes:
-      - redis:/data
-
-  db:
-    image: postgres
-    restart: unless-stopped
-    volumes:
-      - db:/var/lib/postgresql/data
-    networks:
-      - default
-    env_file:
-      - db.env
-
+version: '3'
+
+volumes:
+  db:
+  redis:
+  cert_cache:
+  soundboard:
+
+networks:
+  default:
+
+services:
+  app:
+    build:
+      # We change context so that we can copy the local repo in during
+      # development
+      context: ../
+      dockerfile: yagpdb_docker/Dockerfile
+    restart: unless-stopped
+    command:
```

Comment:
The Dockerfile used here is different than the one used in the hub image so this is invalid as it would run `-all -pa -exthttps=false -https=true` without any executable